components/member-card/index.js

```diff
@@ -1,47 +1,86 @@
 
 const Card = ({ developerInfo }) => {
   const { username, first_name, last_name, img_url, isMember } = developerInfo;
   const socialMedia = ['twitter_id', 'github_id', 'linkedin_id', 'instagram_id'];
   const fullName = `${first_name + ' ' + last_name}`;
 
+  const [showModal, setShowModal] = useState(false);
+
   const brokenImageHandler = (e) => {
     e.target.src = '/images/Avatar.png';
   };
 
+  const showModalHandler = (e) => {
+    e.preventDefault();
+    setShowModal((prevState) => !prevState);
+  };
+
   return (
-    <div>
-      <motion.img
-        layoutId={username}
-        src={img_url + `?${Math.random() * 100}`}
-        onError={brokenImageHandler}
-        className={isMember ? classNames.imgContainer : classNames.imgContainerNewMember}
-        alt={username}
-      />
-      <h2 className={isMember ? classNames.nameOfPerson : classNames.nameOfPersonForNewMember}>
-        {fullName.length > 1 && last_name !== undefined
-          ? fullName.length > 20
-            ? first_name
-            : fullName
-          : username}
-      </h2>
-      {isMember && (
-        <div className={classNames.iconsContainer}>
-          {socialMedia.map(
-            (ele) =>
-              developerInfo[ele] && (
-                <React.Fragment key={ele}>
-                  <SocialMediaIcon id={developerInfo[ele]} type={ele} />
-                </React.Fragment>
-              )
-          )}
-        </div>
-      )}
-    </div>
+    <>
+      <div>
+        <motion.img
+          layoutId={username}
+          src={img_url + `?${Math.random() * 100}`}
+          onError={brokenImageHandler}
+          className={isMember ? classNames.imgContainer : classNames.imgContainerNewMember}
+          alt={username}
+        />
+        <h2 className={isMember ? classNames.nameOfPerson : classNames.nameOfPersonForNewMember}>
+          {fullName.length > 1 && last_name !== undefined
+            ? fullName.length > 20
+              ? first_name
+              : fullName
+            : username}
+        </h2>
+        {isMember && (
+          <div className={classNames.iconsContainer}>
+            {socialMedia.map(
+              (ele) =>
+                developerInfo[ele] && (
+                  <React.Fragment key={ele}>
+                    <SocialMediaIcon id={developerInfo[ele]} type={ele} />
+                  </React.Fragment>
+                )
+            )}
+          </div>
+        )}
+        <button
+          className={classNames.statusIcon}
+          aria-label="Settings Icon"
+          title="Show Modal"
+          onClick={showModalHandler}>
+          <span role="img" aria-label="Settings Icon">
+            ⚙️
+          </span>
+        </button>
+      </div>
+
+      {showModal ? (
+        <Modal show={showModal} hide={showModalHandler} color="" name={username}>
+          <Header hide={showModalHandler}>{username}</Header>
+          <div onClick={(e) => e.stopPropagation()} aria-hidden="true">
```

Comment:
If we have an handler for select , we don't need to stop Propagation here, as we can prevent bubbling with `e.preventDefault()`.
We would be expecting an operation to happen on select right...
You can look into that